modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/handlers/FinishTxCommandHandler.java

```diff
@@ -71,6 +75,15 @@ protected CommandResult handleInternally(
             long commandIndex,
             long commandTerm,
             @Nullable HybridTimestamp safeTimestamp
+    ) throws IgniteInternalException {
+        return handleInternally(command, commandIndex, commandTerm, enlistedPartitions(command));
+    }
+
+    private CommandResult handleInternally(
```

Comment:
This method has only a single use, why did you introduce it? You could have introduced a single variable instead